Utils/old_content_branch.py

```diff
@@ -0,0 +1,169 @@
+
+import argparse
+import os
+import click
+import yaml
+import io
+import json
+from pkg_resources import parse_version
+
+
+def get_file(method, file_path, type_of_file):
+    data_dictionary = None
+    with open(os.path.expanduser(file_path), mode="r", encoding="utf8") as f:
+        if file_path.endswith(type_of_file):
+            read_file = f.read()
+            replaced = read_file.replace("simple: =", "simple: '='")
+            # revert str to stream for loader
+            stream = io.StringIO(replaced)
+            try:
+                data_dictionary = method(stream)
+            except Exception as e:
+                click.secho("{} has a structure issue of file type{}. Error was: {}".format(file_path, type_of_file, str(e)), fg="red")
+                return {}
+    if type(data_dictionary) is dict:
+        return data_dictionary
+    return {}
+
+
+def get_yaml(file_path):
+    return get_file(yaml.safe_load, file_path, ('yml', 'yaml'))
+
+
+def get_json(file_path):
+    return get_file(json.load, file_path, 'json')
+
+
+def handle_yml_file(file_path, new_to_version, should_rewrite=True):
+    yml_content = get_yaml(file_path)
+    if parse_version(yml_content.get('toversion', '99.99.99')) < parse_version(new_to_version) or \
+            parse_version(yml_content.get('fromversion', '0.0.0')) > parse_version(new_to_version):
+        return False
+
+    yml_content['toversion'] = new_to_version
+    if should_rewrite:
+        with open(file_path, 'w') as f:
+            yaml.dump(yml_content, f)
+            print(f" - Updating {file_path}")
+
+    return True
+
+
+def handle_json_file(file_path, new_to_version, should_rewrite=True):
```

Comment:
why does it return true/false? seems weird

Reply:
I'll change it to json_should_stay - it is a checking function and I'll separate the rewrite to a different function -that way it will both look cleaner and be more testable. (same for yml)